src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -10,7 +10,8 @@ namespace ApiView
     public class CodeFileRenderer
     {
         public static CodeFileRenderer Instance = new CodeFileRenderer();
-
+        const string DOCUMENTATION_SPAN_START = "<span class=\"documentation\">";
```

Comment:
```suggestion
        private const string DOCUMENTATION_SPAN_START = "<span class=\"documentation\">";
```

And we typically put consts before other members.